### PR TITLE
Update oxseoencoder.php

### PR DESCRIPTION
**Pull request by TigerLeon**
A little shorter code.

### CODE REVIEW

**Comment by MantasVaitkunas** [comment]
Hi @TigerLeon,
I would suggest you to use prepared statements like this:
```php
$sQ = "SELECT `oxseourl` FROM `oxseo` WHERE `oxstdurl` = ? AND `oxlang` = ? AND `oxshopid` = ? LIMIT 1";

return $oDb->getOne($sQ, array($sStdUrl, $iLanguage, $sShopId));
```
And also to remove ``$sSeoUrl = false;``

Cheers!

**Comment by ooxi** [comment]
Aodblite offers prepared statements? I never realized that O_o

**Comment by MantasVaitkunas** [comment]
@TigerLeon, did you see my last post? Nonetheless I'd like to merge your commit and change it the way I proposed but still your signed OCA is missing. Could you please catch this up? Cheers!

File: source/core/oxseoencoder.php [hunk 1]
```diff
@@ -1232,13 +1232,9 @@ public function fetchSeoUrl($sStdUrl, $iLanguage = null)
...
 
-        $sQ = "SELECT `oxseourl`, `oxlang` FROM `oxseo` WHERE `oxstdurl` = " . $oDb->quote($sStdUrl) . " AND `oxlang` = '$iLanguage' AND `oxshopid` = '$sShopId' LIMIT 1";
-
-        $oDb = oxDb::getDb(oxDb::FETCH_MODE_ASSOC);
-        $oRs = $oDb->select($sQ);
-
-        if (!$oRs->EOF) {
-            $sSeoUrl = $oRs->fields['oxseourl'];
+        $sQ = "SELECT `oxseourl` FROM `oxseo` WHERE `oxstdurl` = " . $oDb->quote($sStdUrl) . " AND `oxlang` = '$iLanguage' AND `oxshopid` = '$sShopId' LIMIT 1";
+        if($sOxseourl = $oDb->getOne($sQ)) {
```
Comment (keywan-ghadami-oxid) [inline source/core/oxseoencoder.php]: be aware that empty string "" and "0" will than not be valid values for oxseourl anymore .

**Comment by Sieg** [comment]
Please head back when you want to discuss about this PR again. Closing it for now.

**Comment by Sieg** [comment]
Hello @TigerLeon, 
Thanks. We got your CLA submission. Will merge your PR soon.

**Comment by MantasVaitkunas** [comment]
@TigerLeon,
Needs code improvement. Maybe change getOne to getRow to ensure the empty string will be validated earlier. See Keywan's comment as well.
Do you want to have a second look at it yourself?

**Comment by TigerLeon** [comment]
The col 'oxseourl' shouldn't be empty. Otherwise the startpage can be broken in some cases (e.g. subshop). So a check for an empty string is not necessary. And a page "domain.com/0/" isn't really a valide page, right?

**Pull request closed without merging**